yw_website/yw_website/urls.py

```diff
@@ -44,13 +44,14 @@
     ###
     path('', yw_website_views.home, name='home'),
     path('my_workflows/', yw_website_views.my_workflows, name='my_workflows'),
-    path('detailed_workflow/<int:document_id>', yw_website_views.detailed_workflow, name='detailed_workflow'),
+    path('detailed_workflow/<int:document_id>/', yw_website_views.detailed_workflow, name='detailed_workflow'),
     path('run_detail/', yw_website_views.run_detail, name='run_detail'),
 
     ###
     # YESWORKFLOW SAVE PATHS
     ###
     path('save/', yw_rest_services_views.create_workflow, name='create'),
+    path('save/<int:workflow_id>/', yw_rest_services_views.update_workflow, name='update'),
```

Comment:
probably better if you remove the trailing `/`

Reply:
in apps i've seen/worked with that was the case. Since django dev's prefer trailing `/`, lets go with that preference.